source/includes/_cluster.md

```diff
@@ -0,0 +1,419 @@
+client.cluster.nodes.list()
+```
+
+```shell
+curl -u user:password http://<manager-ip>/api/v3/cluster/nodes
+```
+
+```javascript
+var headers = {
+   'content-type': 'application/json',
+   'authorization': 'Basic ' + new Buffer(username + ':' + password).toString('base64')
+}
+
+var settings = {
+  "url": "http://<manager-ip>/api/v3/cluster/nodes",
+  "method": "GET",
+  "headers": headers
+}
+
+$.ajax(settings).done(function (response) {
+  console.log(response);
+});
+```
+
+```html
+obsolete
+```
+
+> Response Example
+```json
+{
+    "items":
+    [
+        {
+            "initialized": true,
+            "online": true,
+            "master": true,
+            "host_ip": "172.20.0.2",
+            "name": "cloudify_manager_LMJZA2"
+        }
+    ]
+}
+```
+
+`GET "{manager-ip}/api/v3/cluster/nodes"`
+
+List all nodes in the cluster.
+
+### Response
+
+Field | Type | Description
+----- | ---- | -----------
+`items` | list | A list of `ClusterNode` resources
+
+
+## Get Cluster Node
+
+> Request Example
+```python
+client.cluster.nodes.get("<node-id>")
+```
+
+```shell
+curl -u user:password http://<manager-ip>/api/v3/cluster/nodes/<node-id>
+```
+
+```javascript
+var headers = {
+   'content-type': 'application/json',
+   'authorization': 'Basic ' + new Buffer(username + ':' + password).toString('base64')
+}
+
+var settings = {
+  "url": "http://<manager-ip>/api/v3/cluster/nodes/<node-id>",
+  "method": "GET",
+  "headers": headers
+}
+
+$.ajax(settings).done(function (response) {
+  console.log(response);
+});
+```
+
+```html
+obsolete
+```
+
+> Response Example
+```json
+{
+    "initialized": true,
+    "online": true,
+    "master": true,
+    "host_ip": "172.20.0.2",
+    "name": "cloudify_manager_LMJZA2"
+}
+```
+
+`GET "{manager-ip}/api/v3/cluster/nodes/{node-id}"`
+
+Get the details of a node in the cluster.
+
+### URI Parameters
+* `node-id`: The ID of the node to remove from the cluster
+
+### Response
+
+A `ClusterNode` resource.
+
+
+## Delete Cluster Node
+
+> Request Example
+
+```python
+client.cluster.nodes.delete("<node-id>")
+```
+
+```shell
+curl -X DELETE -u user:password http://<manager-ip>/api/v3/cluster/nodes/<node-id>
+```
+
+```javascript
+var headers = {
+   'content-type': 'application/json',
+   'authorization': 'Basic ' + new Buffer(username + ':' + password).toString('base64')
+}
+
+var settings = {
+  "url": "http://<manager-ip>/api/v3/cluster/nodes/<node-id>",
+  "method": "DELETE",
+  "headers": headers
+}
+
+$.ajax(settings).done(function (response) {
+  console.log(response);
+});
+```
+
+```html
+obsolete
+```
+
+> Response Example
+```json
+{
+    "initialized": true,
+    "online": true,
+    "master": true,
+    "host_ip": "172.20.0.2",
+    "name": "cloudify_manager_LMJZA2"
+}
+```
+
+`DELETE "{manager-ip}/api/v3/cluster/nodes/{node-id}"`
+
+Remove a node from the cluster. The node will disconnect from the cluster and
+disable all cluster mechanisms. It is not possible to rejoin it to the cluster.
```

Comment:
~The node will disconnect from the cluster and disable all cluster mechanisms. It is not possible to rejoin it to the cluster.~
**The node disconnects from the cluster and disables all cluster mechanisms. You cannot rejoin it to the cluster.**